### PR TITLE
YA

### PR DESCRIPTION
**Pull request by Yuhans**
How can i resolve a conflict? You have already merged branches, but now i have to change Task_12, and it's don't want to be merged.:(

### CODE REVIEW

**Comment by Yuhans** [comment]
Well, it seems that i find the answer. :)

File: src/main/java/com/epam/javacourses2016/task14/MyCollection.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package com.epam.javacourses2016.task14;
+
+import java.util.*;
+
+public class MyCollection<T extends Number> implements AbstractCollectionCreator.NumberCollection<T> {
```
Comment (nkarulin) [inline src/main/java/com/epam/javacourses2016/task14/MyCollection.java:5]: Think about extending basic collection, not agregating

**Comment by nkarulin** [comment]
Don't you forget to enable tests on tasks you solved?

**Comment by nkarulin** [comment]
master was massively merged into your remote working branch :gift:

**Comment by Yuhans** [comment]
Looks not good, can you give me advice what should i do now?

**Comment by nkarulin** [comment]
What exactly looks not good? Just resolve those little conflicts and everything must be fine.
Actually it looks like git show more diff then it is. All these test are in your branch now

**Comment by nkarulin** [comment]
:+1:

**Pull request merged**